content/blog/2013/2013-02-15-jenkins-ci-the-origins-of-butlers-build-masters-and-bowties.html

```diff
@@ -1,6 +1,6 @@
 ---
 :layout: post
-:title: 'Jenkins CI: The Origins of Butlers, Build Masters and Bowties'
+:title: 'Jenkins CI: The Origins of Butlers, Build Controllers and Bowties'
```

Comment:
I think this is another term referring to a person, e.g., "Build Master" as a title.